sdks/python/apache_beam/io/gcp/gcsio.py

```diff
@@ -234,7 +271,8 @@ def delete_batch(self, paths):
       request = storage.StorageObjectsDeleteRequest(
           bucket=bucket, object=object_path)
       batch_request.Add(self.client.objects, 'Delete', request)
-    api_calls = batch_request.Execute(self.client._http)  # pylint: disable=protected-access
+    api_calls = batch_request.Execute(self.client._http) # pylint:
+    # disable=protected-access
```

Comment:
<!--new_thread; commit:4b6f31a090cec94795242c2adec791448acc5f0d; resolved:0-->
For pylint overrides on a specific line, it is ok (and preferable) to keep the comment on the same line, even if it exceeds the preferred line length (please revert).